README.md

```diff
@@ -40,25 +40,22 @@ See the [Installation](https://docs.all-hands.dev/modules/usage/installation) gu
 system requirements and more information.
 
 ```bash
-export WORKSPACE_BASE=$(pwd)/workspace
+docker pull docker.all-hands.dev/all-hands-ai/runtime:0.11-nikolaik
 
-docker pull ghcr.io/all-hands-ai/runtime:0.11-nikolaik
-
-docker run -it --pull=always \
-    -e SANDBOX_RUNTIME_CONTAINER_IMAGE=ghcr.io/all-hands-ai/runtime:0.11-nikolaik \
-    -e SANDBOX_USER_ID=$(id -u) \
-    -e WORKSPACE_MOUNT_PATH=$WORKSPACE_BASE \
-    -v $WORKSPACE_BASE:/opt/workspace_base \
+docker run -it --rm --pull=always \
```

Comment:
Removes the openhands app image?

Reply:
`--rm` removes the _container_, instead of keeping it around in `stopped` state (you hit some bugs with these containers getting left around for a long time IIRC)